client/app/services/auth.js

```diff
@@ -55,6 +55,7 @@ export const Auth = {
     window.location.href = `${AuthUrls.Login}?next=${next}`;
   },
   logout() {
+    localStorage.removeItem("recents");
```

Comment:
This should also be in the service. It's better to keep implementation details in the single place and do not expose them